src/ontology/fbbt-edit.obo

```diff
@@ -118583,6 +118583,139 @@ intersection_of: expresses http://flybase.org/reports/FBgn0031903 ! Wnt10
+subset: cur
+intersection_of: FBbt:00058028 ! myosuppressin-positive neuron
+intersection_of: has_soma_location FBbt:00003700 ! adult pars intercerebralis
+relationship: expresses http://flybase.org/reports/FBgn0000546 ! EcR
+relationship: expresses http://flybase.org/reports/FBgn0003255 ! rk
+relationship: fasciculates_with FBbt:00004050 ! crop nerve
+relationship: RO:0002360 FBbt:00110639 ! has dendrite location adult subesophageal zone
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T11:27:10Z
+
+[Term]
+id: FBbt:00058025
+name: adult DMS cell of hypocerebral ganglion
+def: "A dromyosuppressin-positive neuron of the hypocerebral ganglion. There are approximately 5 of these cells. They project to the crop and exert a relaxing activity on the crop muscle (Hadjieconomou et al., 2020)." [FlyBase:FBrf0247320]
+subset: cur
+intersection_of: FBbt:00058028 ! myosuppressin-positive neuron
+intersection_of: has_soma_location FBbt:00004047 ! adult hypocerebral ganglion
+relationship: fasciculates_with FBbt:00004050 ! crop nerve
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T11:35:54Z
+
+[Term]
+id: FBbt:00058026
+name: adult mechanosensory neuron of the hypocerebral ganglion
+def: "A mechanosensory neuron expressing the mechanosensitive ion channel Piezo and innervating the anterior gut. There are approximately 6 of these neurons in the hypocerebral ganglion. They respond to crop-distending stimuli and control feeding behavior (Min et al., 2021)." [FlyBase:FBrf0248237]
+subset: cur
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0264953 ! Piezo
+intersection_of: has_soma_location FBbt:00004047 ! adult hypocerebral ganglion
+relationship: capable_of GO:0060259 ! regulation of feeding behavior
+relationship: fasciculates_with FBbt:00100332 ! adult recurrent nerve
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T16:36:20Z
+
+[Term]
+id: FBbt:00058027
+name: adult chemosensory neuron of the hypocerebral ganglion
+def: "A chemosensory neuron in the hypocerebral ganglion that expresses the fructose receptor Gr43a. There are 4 or 5 of these neurons. They all extend dendritic processes to the foregut lumen, and axonal processes either to the suboesophageal zone or to the musculature of the midgut (Miyamoto et al., 2012; Min et al., 2021)." [FlyBase:FBrf0219961, FlyBase:FBrf0248237]
+subset: cur
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0041243 ! Gr43a
+intersection_of: has_soma_location FBbt:00004047 ! adult hypocerebral ganglion
+relationship: capable_of GO:0050916 ! sensory perception of sweet taste
```

Comment:
Close - you want 'detection of chemical stimulus involved in sensory perception of sweet taste'.

This could go in your logical definition if you wanted to use 'adult sweet taste neuron of the hypocerebral ganglion' or similar as the name